docs/how-to/enable-nts.md

```diff
@@ -0,0 +1,43 @@
+# How to: Enable NTS (Network Time Security)
```

Comment:
I would rephrase this title as `# How to enable Network Time Security`

At the very least, I recommend removing the colon from the title

Reply:
Updated, thanks!